src/integrations/prefect-dask/prefect_dask/task_runners.py

```diff
@@ -119,6 +119,17 @@ class PrefectDaskFuture(PrefectWrappedFuture[R, distributed.Future]):
     def wait(self, timeout: Optional[float] = None) -> None:
         try:
             result = self._wrapped_future.result(timeout=timeout)
+        except (
+            distributed.scheduler.KilledWorker,
+            distributed.comm.CommClosedError,
+        ) as exc:
+            # handle dask specific exceptions that indicate a problem with this task
+            self._final_state = State(
+                type="CRASHED",
+                data=exc,
+                message=f"Encountered an unexpected exception from dask: {exc!r}",
+            )
```

Comment:
Do you think we could update the task state here so that the task shows as CRASHED in the UI?

Reply:
What's the best way to do that -- `emit_event`?

Reply:
I think `prefect.utilities.engine.emit_task_state_change_event` is the easiest way to do that.